pages/docs/reference/whatsnew1420.md

```diff
@@ -0,0 +1,348 @@
+ 
+There are two tolerance policies for this mode:
+- `SEMANTIC`: the compiler will accept code which is syntactically correct, but doesn't make sense semantically,
+    such as `val x: String = 3`.
+
+- `SYNTAX`: the compiler will accept any code, even if it contains syntax errors.
+
+To allow compilation with errors, add the `-Xerror-tolerance-policy=` compiler option with one of the values listed above.
+
+Learn more about ignoring compilation errors with Kotlin/JS IR compiler [here](js-ir-compiler.html#ignoring-compilation-errors).
+
+[**Back to top**](#)
+
+## Kotlin/Native
+
+### Escape analysis
+
+> The escape analysis mechanism is [experimental](evolution/components-stability.html) in 1.4.20. It may change incompatibly in future versions.
+{:.note}
+
+Kotlin/Native receives a prototype of the new [escape analysis](https://en.wikipedia.org/wiki/Escape_analysis) mechanism.
+It improves the runtime performance by allocating certain objects on the stack instead of the heap. With this mecahnism, 
+we've got a 10% average performance increase on our benchmarks, and we continue improving it so that it speeds up the
+program even more.
+
+The escape analysis is performed in a separate compilation phase for the release builds (with the `-opt` compiler option). 
+
+If you want to disable the escape analysis phase, use the `-Xdisable-phases=EscapeAnalysis` compiler option.
+
+### Opt-in wrapping of Objective-C exceptions
+
+> The Objective-C exception wrapping mechanism is [experimental](evolution/components-stability.html) in 1.4.20. It may change incompatibly in future versions.
+{:.note}
+
+Kotlin/Native now can handle exceptions thrown from Objective-C code in runtime to avoid program crashes.
+
+You can opt into wrapping `NSException`’s into Kotlin’s `ForeignException`’s for further handling in the Kotlin code.
+Such a `ForeignExeption` holds the reference to the original `NSException`, which lets you get the information about the root cause.
+
+To enable wrapping of Objective-C exceptions, specify the `-Xforeign-exception-mode objc-wrap` option in the `cinterop`
+call or add `foreignExceptionMode = objc-wrap` property to `.def` file. If you use [CocoaPods integration](native/cocoapods.html),
+specify the option in the `pod {}` build script block of a dependency like this:
+
+```kotlin
+pod("foo") {
+   extraOpts = listOf("-Xforeign-exception-mode”, “objc-wrap")
+}
+```
+
+The default behavior remains unchanged: the program terminates when an exception is thrown from the Objective-C code.
+
+### Support for Xcode 12 libraries
+    
+We have added support for new libraries delivered with Xcode 12. Now you can use them from the Kotlin code.
+
+### Performance improvements and bug fixes
+
+As usual, Kotlin/Native receives performance improvements and bug fixes in various components, including the ones added
+in 1.4.0, for example, the [code sharing mechanism](mpp-share-on-platforms.html#share-code-on-similar-platforms). 
```

Comment:
I don't know how much value adds this example. Don't you want to add a link to a list of resolved tickets in this area?
`As usual` doesn't add much value as well.